presto-benchto-benchmarks/src/main/resources/sql/presto/distributed_sort/sort_query_1_col.sql

```diff
@@ -0,0 +1,6 @@
+SELECT
+  count(orderkey)
+FROM (
+    SELECT *
+    FROM ${database}.${schema}.lineitem
+    ORDER BY orderkey)
```

Comment:
I feel like this is not a practical example. The optimizer should remove sorting from this query altogether. Once it does, this benchmark won't be useful anymore. Consider using a query with a window function: `select *, row_number() over (order by orderkey) from lineitem`. Ditto for a query in `sort_query_6_cols.sql`.

Reply:
I think practical use case is doing insert on top of distributed `ORDER BY`. However, for the benchmarking purposes I wanted to have just read path. Unpartitioned window with `order by` is not rewritten on top of distributed order by yet.

I could either use blackhole connector for inserting or use some other operator (instead of aggregation) that will prevent from removing `ORDER BY`.

Alternatively, this could be addressed if it becomes a problem (`ORDER BY` gets removed) in next benchmarking cycle. For now maybe let's keep the queries as they are since those are the queries for which we have benchmarking results.